### PR TITLE
Try installing ifort on windows

### PR DESCRIPTION
**Pull request by scharlton2**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by scharlton2** [comment]
COMPONENTS

**Comment by scharlton2** [comment]
@conda-forge-admin, please rerender

**Comment by scharlton2** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,13 +10,11 @@ source:
   sha256: a737521126efc3295b818acc951a745d7b914446836c7931b0b52998f3934e2d
 
 build:
-  number: 0
-  skip: true  # [win]
+  number: 1
+  skip: true  # [not win]
```
Comment (jdhughes-dev) [inline recipe/meta.yaml]: Does this need to be modified to build non-windows versions as well. I only see the Azure windows build.

Comment (scharlton2) [reply recipe/meta.yaml]: Yes, I'm just testing the windows part right now.  (I didn't want to waste a bunch of resources on the other builds).  @jdhughes-usgs do you know if the meson build for intel fortran is statically linked, or does it need the intel fortran runtime dlls?

Comment (jdhughes-dev) [reply recipe/meta.yaml]: The meson build will be statically linked

**Comment by scharlton2** [comment]
@conda-forge-admin, please rerender

**Comment by scharlton2** [comment]
@conda-forge-admin, please rerender

**Comment by scharlton2** [comment]
Just searched the conda-forge groups and found a feedstock doing the same thing for building on windows:

https://github.com/conda-forge/openfast-feedstock

https://github.com/conda-forge/openfast-feedstock/blob/main/recipe/bld.bat

So, I'm much less hesitant to merge this now.  What do you think @jdhughes-usgs?  You might want to look at the changes I made to the meson options. (I changed debug to false and removed the optimization=0 setting)

**Comment by jdhughes-dev** [comment]
> Just searched the conda-forge groups and found a feedstock doing the same thing for building on windows:
> 
> https://github.com/conda-forge/openfast-feedstock
> 
> https://github.com/conda-forge/openfast-feedstock/blob/main/recipe/bld.bat
> 
> So, I'm much less hesitant to merge this now. What do you think @jdhughes-usgs? You might want to look at the changes I made to the meson options. (I changed debug to false and removed the optimization=0 setting)

@scharlton2 The meson changes are good - they are consistent with how we compile on CI. I think we should merge the PR.

**Comment by scharlton2** [comment]
@jdhughes-usgs I think you're going to have to merge it.  I don't think I have the power yet.

**Pull request merged**